docs/rule/require-each-key.md

```diff
@@ -0,0 +1,54 @@
+# require-each-key
+
+You might be having huge performance issues when rendering large data-sets with `{{#each}}`
+
+In order to avoid that, it is recommended to use `key` with `{{#each}}` because, Ember will not rerendering all items, but only the ones that actually needed to be updated.
```

Comment:
> because, Ember will not rerendering all items, but only the ones that actually needed to be updated.

Something is wrong with wording and punctuation here.

This sounds better to me:

> so that Ember would not rerender all items, but only the ones that actually need to be updated.

But better to consult with a native English speaker :)

Reply:
Yes, I agree with you, this needs to have better wording, @MelSumner Can you please help?